packages/p2p/src/components/order-details/payment-method-accordion-header.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { Icon, Text } from '@deriv/components';
+import './payment-method-accordion-header.scss';
 
 const PaymentMethodAccordionHeader = ({ payment_method }) => {
     const method = payment_method.display_name.replace(/\s|-/gm, '');
 
     if (method === 'BankTransfer' || method === 'Other') {
         return (
-            <div className='order-details-card__accordion-row'>
-                <Icon className='order-details-card__accordion-icon' icon={`IcCashier${method}`} size={16} />
-                <div className='order-details-card__accordion-title'>
+            <div className='payment-method-accordion-header-row'>
```

Comment:
shouldn't these be payment-method-accordion-header__row and payment-method-accordion-header__icon, __title, __column, etc. ?